remove moment js

radekrepka opened the pull request.

jesuscc1993 committed:
Added selectedDay state update on selected prop change.

jesuscc1993 committed:
Applied requested fix

jesuscc1993 committed:
Merge branch 'master' of https://github.com/mimacom/react-native-calendars into feature/update-selected-day-on-prop-change

jesuscc1993 committed:
Fixed bad merge.

jesuscc1993 committed:
Reverted automatic formatting.

jesuscc1993 committed:
Merge branch 'master' into feature/update-selected-day-on-prop-change

jesuscc1993 committed:
Update index.js

Fixed argument name.

jesuscc1993 committed:
Merge branch 'master' into feature/update-selected-day-on-prop-change

jesuscc1993 committed:
Fixed date comparison on componentDidUpdate function

jesuscc1993 committed:
Removed leading spaces in empty line

jesuscc1993 committed:
Removed spaces next to curly brackets

yoangau committed:
fix 'calendar-list' not rendering on web

Commit:
remove usage of ViewPropTypes

Inbal-Tish committed:
AgendaList - 'blockDateUpdate' to block updates to the CalendarContextProvider when list scrolls

Inbal-Tish committed:
renaming prop

Commit:
make moment js as an optional dependency

Inbal-Tish committed:
Changing prop name to `avoidDateUpdates`

petrkonecny2 reviewed (changes requested):
I don't like this. At all. Change my mind :)

Inbal-Tish committed:
Merge pull request #1441 from wix/1.1255.0/AgendaList_blockDateUpdates

AgendaList - block date updates

ethanshar committed:
Fix issue with reversed scroll offset values on Android RTL

jodrescher committed:
Update index.js

fixed missing horizontal option, otherwise "horizontal" is always undefined in CalendarListItem

Inbal-Tish committed:
Merge pull request #1449 from wix/fix/ExpandableCalOnAndroidRTL

Fix issue with reversed scroll offset values on Android RTL

Inbal-Tish committed:
Edit changelog for version 1.1255.0

Inbal-Tish committed:
ExpandableCalendar - fix static header updates when pressing header arrows

Inbal-Tish committed:
Merge pull request #1466 from wix/ExpandableCalendar_fix_staticHeader_updates

ExpandableCalendar - fix static header updates

Inbal-Tish committed:
Edit CHANGELOG for version 1.1256.0

Inbal-Tish committed:
test

Inbal-Tish committed:
Merge pull request #1424 from bananer/remove_viewproptypes

remove usage of ViewPropTypes

Inbal-Tish committed:
Update changelog for new version (1.1256.0).

Inbal-Tish committed:
Merge pull request #1405 from yoangau/fix-calendar-list-not-rendering

fix 'calendar-list' not rendering on web

Inbal-Tish committed:
Merge pull request #1451 from jodrescher/month-arrow-horizontal-bug

Fixed undefined horizontal prop in CalendarListItem

Inbal-Tish committed:
Merge pull request #1080 from mimacom/feature/update-selected-day-on-prop-change

Update selected date in state when prop changes

Inbal-Tish committed:
edit changelog - version 1.1256.0 release

Inbal-Tish committed:
version 1.1257.0 - edit changelog

radekrepka committed:
Merge branch 'master' into remove-moment
